.github/workflows/lint.yml

```diff
@@ -0,0 +1,38 @@
+name: Run linting
+
+on:
+    # Run on all pushes and on all pull requests.
+    # Prevent the "push" build from running when there are only irrelevant changes.
+    push:
+        paths-ignore:
+            - "**.md"
+    pull_request:
+    # Allow manually triggering the workflow.
+    workflow_dispatch:
+
+jobs:
+    test:
+        name: Run lint
+        # Ubuntu-20.x includes MySQL 8.0, which causes `caching_sha2_password` issues with PHP < 7.4
+        # https://www.php.net/manual/en/mysqli.requirements.php
+        # TODO: change to ubuntu-latest when we no longer support PHP < 7.4
+        runs-on: ubuntu-18.04
+
+        steps:
+            - name: Checkout code
+              uses: actions/checkout@v2
+
+            - name: Install node
+              uses: actions/setup-node@v2
+              with:
+                  node-version: "14"
+
+            - name: NPM install
+              run: npm install
+
+            - name: Run lint-scripts
+              run: npm run lint-scripts
+
+            - name: Run lint-styles
+              if: ${{ always() }}
```

Comment:
What does this line add?

Reply:
ensures that `run lint-styles` will run even if the preceding step fails.

Reply:
Would a [`continue-on-error`](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error) on the previous step be a better approach, as then each and every potential future step doesn't need to include the `if` condition.

Reply:
`continue-on-error` leads to a green checkmark which I think would be misleading https://github.com/Automattic/liveblog/runs/2906734901?check_suite_focus=true

vs the if-always, which will go red while also giving us the results of both steps https://github.com/Automattic/liveblog/pull/674/checks?check_run_id=2906654064

Reply:
Huh - it's even [documented](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#job-status-check-functions) as that. Makes sense.